src/animation/AnimationClip.d.ts

```diff
@@ -45,6 +45,6 @@ export class AnimationClip {
 		animation: any,
 		bones: Bone[]
 	): AnimationClip;
-	static toJSON(): any;
+	static toJSON( json: any ): any;
```

Comment:
I should be:
```ts
static toJSON( clip: AnimationClip ): any;

Reply:
Yes, I made a mistake.

Reply:
Don't worry. Bob Ross would have said: _We don't make mistakes, just happy little accidents._ 😊